website/docs/plugin/framework/migrating/attributes-blocks/blocks-computed.mdx

```diff
@@ -0,0 +1,123 @@
+---
+page_title: 'Computed Blocks: Migrating from SDKv2 to the Framework'
+description: >-
+  Migrate computed blocks from SDKv2 to attribute validators in the plugin Framework.
+---
+
+# Computed Blocks
+
+Some providers, resources, and data sources include repeatable nested blocks in their attributes. Some blocks contain
+fields  with `Computed: true`, which means that the provider code can define the value or that it could come from the
+output of terraform apply (e.g., the ID of an EC2 instance). Refer to [Schemas - Blocks](/plugin/framework/FIXME) in the
+Framework documentation for details.
+
+This page explains how to migrate computed-only blocks from SDKv2 to the Framework. Refer to
+[Blocks](/plugin/framework/migrating/attributes-blocks/blocks) if you are looking for information about migrating blocks
+that are practitioner configurable.
+
+## SDKv2
+
+In SDKv2, blocks are defined by an attribute whose type is `TypeList`, or `TypeSet` and whose `Elem` field is set to a
+`schema.Resource` that contains a map of the block's attribute names to corresponding `schemaSchema` structs.
```

Comment:
```suggestion
In SDKv2, blocks are defined by an attribute whose type is either `TypeList` or `TypeSet`, and whose `Elem` field is set to a
`schema.Resource` that contains a map of the block's attribute names to corresponding `schemaSchema` structs.
```

Is this correct? The original seems to imply that `TypeList` attributes don't need the Elem field set because comma placement.